python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
     num = 0
     for v in vowels:
         num += text.lower().count(v)        
     return num
 
 def num_consonants(text):
```

Comment:
This can be simplfied as a diff of num_vowels
```suggestion
def num_consonants(text):
      vowel_count = num_vowels(text)
      return len(text) - vowel_count

```